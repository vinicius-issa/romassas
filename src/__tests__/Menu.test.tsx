import React from 'react';
import { render } from '@testing-library/react';
import Enzyme, {  mount } from 'enzyme'
import Menu from '../components/elements/Menu';
import Adapter from 'enzyme-adapter-react-16';
 
Enzyme.configure({ adapter: new Adapter() });
describe('Testing Menu Component', ()=>{
    it('should render Menu with Login Button if user is not logged', () => {
        const wrapper = mount(
            <Menu/>
        );
        expect(wrapper.html()).toMatchSnapshot();
    });
});

