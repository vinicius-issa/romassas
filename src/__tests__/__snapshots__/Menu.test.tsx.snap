// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Menu Component should render Menu with Login Button if user is not logged 1`] = `"<header class=\\"MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary MuiPaper-elevation4\\"><div class=\\"MuiToolbar-root MuiToolbar-regular MuiToolbar-gutters\\"><button class=\\"MuiButtonBase-root MuiIconButton-root MuiIconButton-colorInherit MuiIconButton-edgeStart\\" tabindex=\\"0\\" type=\\"button\\" aria-label=\\"menu\\"><span class=\\"MuiIconButton-label\\"><svg class=\\"MuiSvgIcon-root\\" focusable=\\"false\\" viewBox=\\"0 0 24 24\\" aria-hidden=\\"true\\"><path d=\\"M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z\\"></path></svg></span><span class=\\"MuiTouchRipple-root\\"></span></button><h6 class=\\"MuiTypography-root header MuiTypography-h6\\">News</h6><button class=\\"MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-colorInherit\\" tabindex=\\"0\\" type=\\"button\\"><span class=\\"MuiButton-label\\">Login</span><span class=\\"MuiTouchRipple-root\\"></span></button></div></header>"`;
